charts/node-analyzer/README.md

```diff
@@ -88,7 +88,7 @@ The following table lists the configurable parameters of the Sysdig Node Analyze
 | `sysdig.accessKey`                                                   | Your Sysdig Agent Access Key                                                             | ` ` Either accessKey or existingAccessKeySecret is required                    |
 | `sysdig.existingAccessKeySecret`                                     | Alternatively, specify the name of a Kubernetes secret containing an 'access-key' entry  | ` ` Either accessKey or existingAccessKeySecret is required                    |
 | `secure.enabled`                                                     | Enable Sysdig Secure                                                                     | `true`                                                                         |
-| `secure.vulnerabilityManagement.newEngineOnly`                       | Enable only the new vulnerability management engine                                         | `false`                                                                         |
+| `secure.vulnerabilityManagement.newEngineOnly`                       | Enable only the new vulnerability management engine                                         | `false`                                                                        |
```

Comment:
```suggestion
| `secure.vulnerabilityManagement.newEngineOnly`                       | Enable only the new vulnerability management engine                                         | `false`                                                                      |
```
just a nit

Reply:
maybe not accurately visible my above change but I just removed 2 spaces in the end block.